csvqb/csvqb/tests/unit/cube/qb/errorvalidation.py

```diff
@@ -19,7 +19,7 @@ def test_single_measure_qb_definition(self):
             "Value": [2, 2, 2]
         })
 
-        metadata = CubeMetadata(URI("http://example.com/some/dataset"), "Some Dataset")
+        metadata = CubeMetadata("Some Dataset")
```

Comment:
Can you revert this change? I've got a fix for it in another PR and don't want more conflicts than we absolutely need.